lib/ts/logger.ts

```diff
@@ -23,12 +23,14 @@ const SUPERTOKENS_DEBUG_NAMESPACE = "com.supertokens";
 */
 
 function logDebugMessage(message: string) {
-    if (debug.enabled(SUPERTOKENS_DEBUG_NAMESPACE)) {
-        debug(SUPERTOKENS_DEBUG_NAMESPACE)(
-            `{t: "${new Date().toISOString()}", message: \"${message}\", file: \"${getFileLocation()}\" sdkVer: "${version}"}`
-        );
-        console.log();
-    }
+    debug(SUPERTOKENS_DEBUG_NAMESPACE)(
```

Comment:
why aren't you checking for `if (debug.enabled(SUPERTOKENS_DEBUG_NAMESPACE)) {`?

Reply:
We already call `debug.enable()` when initialising the logger, so ideally this should always return `true`. Added this check, just for safety.